src/06_shared/img/svg/check.svg

```diff
@@ -1,5 +1,5 @@
 <svg viewBox="0 0 25 25" xmlns="http://www.w3.org/2000/svg" fill="none">
-<path d="M19 5L8.6875 16L4 11" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M19 5L8.6875 16L4 11" stroke="url(#paint0_linear_176_9088)" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
Need to return black svg. Use mask for making gradient https://www.w3schools.com/css/css3_masking.asp